### ISSUE TITLE
Picasso with placeholder after loaded images from URL is showing the url images, but with placeholder size

### ISSUE THREAD

**Issue by RaghavendhraK** [gtomato/carouselview#1]
Hi,

We are showing the images from the Picasso inside the carousel view, with the below code.
```
Picasso.with(context)
             .load(thumbURL)
             .placeholder(R.drawable.spin)
             .error(R.drawable.image_unavailable)
             .into(holder.slide);
```
The small placeholder image shows first. But after when the image is loaded from URL, the image still shows in the size of small placeholder. But once we go and come back to the same screen(refresh), this time the same image appears in the right size. As the same image loaded this time is from the Memory Cache.

Could you please let us know do we have any fixes/workaround for this one.

Thanks,
Raghu

**Comment by sunny-chung** [comment]
Thanks for using our library and creating the first issue to us!

There is a bug that the layout manager does not measure visible children views again on layout pass. Unfortunately, it seems there is no workaround except modifying the library.

I will fix it and roll out in the next few days. Please stay tuned.

**Comment by sunny-chung** [comment]
Fixed in v2.0.1.
